pyproject.toml

```diff
@@ -31,6 +31,7 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
   "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3.14",
```

Comment:
What are your general thoughts on removing such classifiers?

Reply:
I think it's useful metadata, because they indicate that the maintainer has tested these versions and support them.

If there are no classifiers, I just have to try and install and run and hope for the best.